ssl/ssl_local.h

```diff
@@ -294,8 +294,14 @@
  * flags because it may not be set to correct version yet.
  */
 # define SSL_CLIENT_USE_TLS1_2_CIPHERS(s)        \
```

Comment:
I'd be tempted to remove this macro altogether and just merge its contents into `SSL_CLIENT_USE_SIGALGS()` directly. It's not used anywhere except in that macro.